myuw/templates/handlebars/card/summary/schedule.html

```diff
@@ -2,7 +2,7 @@
     {% tplhandlebars "instructor_summary_schedule" %}
     <div class="myuw-card teaching-schedule-summary" data-type="card"
          data-name="SummaryScheduleCard" data-identifier="{{ title }}">
-         <div style="position:relative;">
+         <h4 style="position:relative;">
```

Comment:
@DannyNineNineNine why did this get changed to an `h4`?

Reply:
That was me - an accident, maybe my IDE?